platforms/hyperledger-fabric/configuration/roles/create/ca-tools/peer/tasks/write.yaml

```diff
@@ -0,0 +1,36 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+---
+############################################################################################
+# This task copy the peer certificate to the path provided in network.yaml
+- name: Copy msp cacerts from auto-generated path to given path
+  shell: |
```

Comment:
This task should be done by pulling the certificate from vault and not using `kubectl cp` cmd